pkg/models/model_studio.go

```diff
@@ -10,6 +10,7 @@ type Studio struct {
 	Checksum  string          `db:"checksum" json:"checksum"`
 	Name      sql.NullString  `db:"name" json:"name"`
 	URL       sql.NullString  `db:"url" json:"url"`
+	ParentId  int             `db:"parent_id" json:"parent_id"`
```

Comment:
Wouldnt  `sql.NullInt64` be more relevant as a type for ParentID ? In  cases with fks (eg scene,gallery) we use `sql.NullInt64` afaik

Reply:
Edit from the  integration test in travis log it seems that the type chage is needed anyway
`panic: Could not populate database: sql: Scan error on column index 7, name "parent_id": converting driver.Value type <nil> ("<nil>") to a int: invalid syntax`
Have a look at the scene code to see how sql.NullInt64 is handled